### ISSUE TITLE
Pointless Snad

### ISSUE THREAD

**Issue by IamStig** [FTBTeam/FTB-Modpack-Issues#2550]
### Modpack

FTB Presents Direwolf20 1.19

### Modpack version

1.1.1

### Describe the bug

Not sure if it is a bug or by design but Snad is included in the pack BUT it seems nerfed. I have placed 2 sand and 2 snad in a row and the cactus (while it does grow tall on either, via another mod I believe) it does not grow any faster. I even tried raising the snad config from 2 to 10 and there is no difference in growth rate.
Has Snad only been include to stop players adding it themselves? 

### Steps to reproduce

seems obvious - see above

### Expected behaviour

While SB3 had it very OP I at least expected to see some speed increase in DW20 pack

### Additional information

I could have included a screenshot of the 4 cactus with 3 at 3 blocks high and the 4th at 4 blocks but I'd shut the game.

**Comment by Yusunoha** [comment]
perhaps this might be because sugarcane, cactus and bamboo are replaced with the apotheosis variant in this pack, and snad might be only coded to work with the vanilla variants.

**Comment by Saereth** [comment]
Yeah it's an apotheosis conflict, looking into it

**Comment by Saereth** [comment]
https://github.com/RoBrit/Snad/issues/25

**Comment by IamStig** [comment]
I loaded the DW20 pack today to find the in world Snad missing, it seems that FTB removed the mod (although I can not find mention of it in the changelog). I added in the only 1.19 version of Snad and the game would not load. 
I am guessing that is due to the DW20 pack being (and unlikely to change from) 1.19.2 and snad being 1.19.3.
Just an FYI, I can understand not wanting to support 1.19.2 and 1.19.3.
